scripts/zones/Abyssea-Konschtat/IDs.lua

```diff
@@ -65,35 +65,30 @@ zones[xi.zone.ABYSSEA_KONSCHTAT] =
     },
     mob =
     {
+        ASHTAERH_THE_GALLVEXED  = 16838718,
```

Comment:
Teo might be able to chime in better for this, but I think the strategy we should employ here should be either name-based or handle in the mob script itself.  For unique names, we could use hidden quests to handle, but that would accomplish the same as handling on the NPC.  Would appreciate thoughts on this topic.

Reply:
I'd actually like to see how we might do abyssea KI drops and NM spawns and such as hidden quests as thy may be a much better pattern, but I don't know how that works exactly to say for sure. Thus far we've been doing `NM_NAME = ID,` (and placeholders for lotto) like this because they **are** unique and its much saner handling for ID shifts to have it all in the IDs.lua than in 50 different QM scripts (because we had automation handling it were intended to expand on).

Actual KI drops themseleves Loki and I used to put right in the NMs onMobDeath script and somewhere after we stopped working on it somebody else globalized that.

Reply:
I think I like that pattern, with perhaps a hidden quest file per-zone.  Then keep the onMobDeath and onTrigger functions there; We'd still need mixins on the mob script itself though, but everything else could be handled